multinode-demo/common.sh

```diff
@@ -59,6 +59,7 @@ solana_wallet=$(solana_program wallet)
 solana_replicator=$(solana_program replicator)
 
 export RUST_BACKTRACE=1
+export SOLANA_METRICS_HIGH_RATE=10
```

Comment:
Don't you want a default of 10 and then set to 1000 here?

Reply:
Hmm I guess I'm not sure 🤔Can you help me understand the different ways a cluster is run? Do you use `multinode-demo` to start your benchmark tests?

Reply:
I just swapped them, but still want to understand to make sure all the scenarios are covered. I'm worried that having a default of `1000` in `multinode-demo` will be surprising to other engineers

Reply:
@jstarry I use both local testing with:
```
./multinode-demo/setup.sh
./multinode-demo/bootstrap-leader.sh
./multinode-demo/client.sh
```

and with a remote cluster with:

`./net/net.sh restart -f cuda`

Reply:
Also like this: `cd bench-tps; cargo test --release bench_tps_local_cluster_solana`

But rarely would i have metrics enabled for that.

Reply:
@sakridge great, thanks for clarifying! Looks like `net.sh` calls into `multinode-demo` scripts so I get now why you want a high default there. I added a small change to make sure the `multinode-demo` scripts won't override `SOLANA_METRICS_HIGH_RATE` if it is already set.